pandas-stubs/core/frame.pyi

```diff
@@ -2070,40 +2070,44 @@ class DataFrame(NDFrame, OpsMixin):
         limit: int | None = ...,
         tolerance=...,
     ) -> DataFrame: ...
+    # Rename axis with `mapper`, `axis`, and `inplace=True`
     @overload
     def rename_axis(
         self,
-        mapper=...,
+        mapper: Scalar | ListLike | None = ...,
```

Comment:
`mapper` was already accepting `None` here, as it was untyped. I've added types matching the [docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.rename_axis.html).

Reply:
ok